### PR TITLE
DOCSP-42336: Cluster monitoring

### PR DESCRIPTION
**Pull request by norareidy**
# Pull Request Info

[PR Reviewing Guidelines](https://github.com/mongodb/docs-scala/blob/master/REVIEWING.md)

JIRA - https://jira.mongodb.org/browse/DOCSP-42336
Staging - https://deploy-preview-73--docs-scala.netlify.app/monitoring/cluster-monitoring/

## Self-Review Checklist

- [ ] Is this free of any warnings or errors in the RST?
- [x] Did you run a spell-check?
- [x] Did you run a grammar-check?
- [ ] Are all the links working?
- [ ] Are the [facets and meta keywords](https://wiki.corp.mongodb.com/display/DE/Docs+Taxonomy) accurate?

### CODE REVIEW

File: source/monitoring/cluster-monitoring.txt [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+     - `ClusterListener <{+core-api+}/com/mongodb/event/ClusterListener.html>`__
+     - Created when the driver first connects to the cluster.
+
+   * - `ClusterClosedEvent <{+core-api+}/com/mongodb/event/ClusterClosedEvent.html>`__
+     - `ClusterListener <{+core-api+}/com/mongodb/event/ClusterListener.html>`__
+     - Created when the driver disconnects from the cluster.
+
+   * - `ServerDescriptionChangedEvent <{+core-api+}/com/mongodb/event/ServerDescriptionChangedEvent.html>`__
+     - `ServerListener <{+core-api+}/com/mongodb/event/ServerListener.html>`__
+     - Created when the server description changes
```
Comment (mcmorisi) [inline source/monitoring/cluster-monitoring.txt:124]:
```suggestion
     - Created when the server description changes.
```

**Review by mcmorisi — APPROVED** [review]
LGTM, one nit!

File: source/includes/monitoring/sdam.scala [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+    override def clusterOpening(event: ClusterOpeningEvent): Unit =
+        println(s"Cluster with ID ${event.getClusterId} opening")
+
+    override def clusterClosed(event: ClusterClosedEvent): Unit =
+        println(s"Cluster with ID ${event.getClusterId} closed")
+
+    override def clusterDescriptionChanged(event: ClusterDescriptionChangedEvent): Unit = {
+        if (!isWritable) {
+        if (event.getNewDescription.hasWritableServer) {
```
Comment (jyemin) [inline source/includes/monitoring/sdam.scala:19]: Let's fix the indentation in this method.

**Review by jyemin — APPROVED** [review]
One comment on the indentation, but once that's addressed LGTM.

**Pull request merged**